plugins/kotlin/uast/uast-kotlin-fir/test/org/jetbrains/kotlin/idea/fir/uast/FirUastApiTest.kt

```diff
@@ -33,6 +33,146 @@ open class FirUastApiTest : AbstractFirUastTest() {
             // Bogus
         }
 
+        @TestMetadata("AnnotationParameters.kt")
```

Comment:
This test suit looks very similar to a generated one but it is not. Maybe it can be generated? :)

Reply:
That's because I (manually) mimic generated tests so that we can easily go to the test input from test run panel. :)

Hm, regarding making it generated, resolve/api tests exercise different surfaces/aspect of APIs per test input; there won't be a unified output; plus, not all test inputs are used. So, I'm not sure what we can get from automation...

Reply:
Got it